models/conversations.go

```diff
@@ -0,0 +1,40 @@
+package models
+
+import (
+	"context"
+)
+
+type Conversation struct {
+	ID      string `json:"id" bson:"_id,omitempty"`
+	GroupID string `json:"groupId" bson:"group_id,omitempty"`
+	Title   string `json:"title" bson:"title,omitempty"`
+}
+
+type ConversationInfo struct {
+	Title   string `json:"title" bson:"title,omitempty"`
+	GroupID string `json:"groupId" bson:"group_id,omitempty"`
+}
+
+type ConversationTitle struct {
```

Comment:
This should be named `UpdateConversationPayload`. What if in the future we want to be able to update more fields of a conversation? Then the naming `ConversationTitle` wouldn't really make sense.